src/Tools/ExternalAccess/Razor/RazorLanguageServerFactoryWrapper.cs

```diff
@@ -21,13 +23,8 @@ internal class RazorLanguageServerFactoryWrapper : IRazorLanguageServerFactoryWr
 
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
         [ImportingConstructor]
-        public RazorLanguageServerFactoryWrapper(ILanguageServerFactory languageServerFactory)
+        public RazorLanguageServerFactoryWrapper(ILanguageServerFactory languageServerFactory!!)
```

Comment:
we shouldn't use this syntax considering latest decisions

Reply:
The !! syntax is being removed

Reply:
😮 missed the memo on this, good to know!